libraries/LittleFS/src/LittleFS.h

```diff
@@ -221,13 +221,48 @@ class LittleFSImpl : public FSImpl
             return false;
         }
 
+        if(_timeCallback && _tryMount()) {
+            // Mounting is required to set attributes
+
+            time_t t = _timeCallback();
+            rc = lfs_setattr(&_lfs, "/", 'c', &t, 8);
+            Serial.printf("rc1: %d\n", rc);
```

Comment:
CAn we get rid of this and the following debug printout, please?

Reply:
You're absolutely right. I'm sorry for leaving these in the PR.